ketcher-autotests/tests/File-Management/Open-And-Save-Files/Image-files/image-files.spec.ts

```diff
@@ -1948,7 +1948,7 @@ test.describe('Image files', () => {
 
   test('Verify that added by Tool SVG images with elements are displayed on preview and can be saved together to PNG file with the correct positions and layers after selection, moving actions of images', async () => {
     /**
-     * Test case: https://github.com/epam/Indigo/issues/2161
+     * Test case: https://github.com/epam/Indigo/issues/2162
```

Comment:
Why did you change the task number?

Reply:
Because the number of task isn't correct